accountsdb-plugin-intf/src/accountsdb_plugin_intf.rs

```diff
@@ -0,0 +1,71 @@
+use std::any::Any;
+use std::io;
+/// The interface for AccountsDb plugins. A plugin must implement
+/// the AccountsDbPlugin trait to work with the Solana Validator.
+/// In addition the dynamic libraray must export a "C" function _create_plugin which
+/// creates the implementation of the plugin.
+use thiserror::Error;
+
+#[derive(Clone, PartialEq, Default, Debug)]
+pub struct ReplicaAccountMeta {
+    pub pubkey: String,
+    pub lamports: u64,
+    pub owner: String,
+    pub executable: bool,
+    pub rent_epoch: u64,
+}
+
+impl Eq for ReplicaAccountInfo {}
+
+#[derive(Clone, Default, PartialEq, Debug)]
+pub struct ReplicaAccountInfo {
+    pub account_meta: ReplicaAccountMeta,
+    pub hash: String,
+    pub data: Vec<u8>,
+}
```

Comment:
Any plans to dedupe with replica-lib?

Reply:
Yes. I am planning to use the 3rd installment of AccountsDb replication PR to address this